client/foundries_sboms.go

```diff
@@ -44,3 +52,32 @@ func (s Sbom) Download(dst string) error {
 	}
 	return os.WriteFile(dst, *buff, 0o744)
 }
+
+func (a *Api) TargetSbomPackages(factory, targetName, path string) ([]SpdxPackage, error) {
+	url := a.serverUrl + "/ota/factories/" + factory + "/targets/" + targetName + "/sboms/" + path
+	logrus.Debugf("TargetSbomPackages with url: %s", url)
+
+	body, err := a.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	type rv struct {
+		Packages []SpdxPackage `json:"packages"`
+	}
+
+	var doc rv
+	if err = json.Unmarshal(*body, &doc); err != nil {
+		return nil, err
+	}
+	return doc.Packages, nil
+}
+
+func (p SpdxPackage) License() string {
+	if p.LicenseConcluded == p.LicenseDeclared {
+		return p.LicenseConcluded
+	} else if p.LicenseConcluded == "NOASSERTION" {
+		return p.LicenseDeclared
+	}
+	panic(fmt.Errorf("Unknown license configuration for package: %s - Concluded(%s) Declared(%s)", p.Name, p.LicenseConcluded, p.LicenseDeclared))
```

Comment:
I'm not sure we need to panic here.
This means that in case of one broken item in SBOM customer cannot view anything.
I'd rather return here something sensible (e.g. a given error message).